projects/driver_controls/test/test_ads1015.c

```diff
@@ -0,0 +1,68 @@
+#include <stdint.h>
+#include <stdio.h>
+#include "ads1015.h"
+#include "gpio_it.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "unity.h"
+
+static Ads1015Storage storage;
+GPIOAddress ready_pin = { GPIO_PORT_B, 2 };
+
+static I2CSettings i2c_settings = {
+  .speed = I2C_SPEED_FAST, .scl = { GPIO_PORT_B, 10 }, .sda = { GPIO_PORT_B, 11 }
```

Comment:
Please fix your formatting and naming - static variables should be prefixed with `s_`.